packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/adding-basic-auth/index.md

```diff
@@ -15,7 +15,9 @@ To add the `npm` packages:
 
 After including the packages, add the following code snippet in your project.
 
-> **Note:** As part of the hook configuration process, you add the authorization field and secret in your Okta org, which inputs this content into the Okta request header. The authorization and secret are authorized by your external service.
+As part of the Inline Hook configuration process, you add the authorization field and secret in your Okta org, which inputs this content into the Okta request header. The authorization and secret are authorized by your external service.
```

Comment:
> As part of the Inline Hook configuration process, you must add the authorization field and [client?] secret to your Okta org, which then adds this content into the Okta request header. The authorization and [client?] secret are authorized by your external service.

Again - this might be a context issue - to what are the "secret" and the "external service" referring?

Reply:
The "secret" is the Authentication secret (as referenced by the Okta org) and is the user:password credentials, rendered in Base64, which are added to the request header.  The external service is the application the inline hook api calls (and authenticates the API call using Basic Auth in the header).

But this is a great comment as I don't address what's happening at a high-level, which I should be at this early point in the doc. Re-wrote those two paragraphs as follows:

The Inline Hook examples use [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication) to authenticate the Okta Inline Hook API calls by your Glitch external service. The user and password credential pair stored in your Glitch application must be encoded in Base64 and added as the Authentication secret when you activate the Inline Hook on your Okta org. Ensure you add the scheme `Basic` to the authentication secret value. 

For example, the credential pair used in the Inline Hook examples is `admin:supersecret`, which encoded in Base64 is `YWRtaW46c3VwZXJzZWNyZXQ= `. Adding the scheme to this value, creates the Inline Hook Authentication secret value: `Basic YWRtaW46c3VwZXJzZWNyZXQ=`.